app/src/main/java/com/example/frogweather/data/Constants.kt

```diff
@@ -21,4 +21,11 @@ const val WIND_DIRECTION_NO_INDICATION = "no indication"
 const val WIND_DIRECTION_ARROWS = "arrows"
 const val WIND_DIRECTION_ABBREVIATIONS = "abbreviations"
 const val NOTIFICATION_TYPE_DEFAULT = "default"
-const val NOTIFICATION_TYPE_SIMPLE = "simple"
\ No newline at end of file
+const val NOTIFICATION_TYPE_SIMPLE = "simple"
+const val WEATHER_API_BASE_URL = "https://api.openweathermap.org"
+const val WEATHER_API_PATH_URL = "/data/2.5/forecast?units=metric"
+const val API_KEY = "721661b090391eed20fcca7c71da55ae"
```

Comment:
Not blocking: API key can be moved to the gradle, just a note

Reply:
done